ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-release-1.24.yaml

```diff
@@ -223,7 +223,7 @@ tests:
           cd /work && \
           entrypoint \
           prow/integ-suite-kind.sh \
-          test.integration.pilot.kube.presubmit"
+          test.integration.pilot.kube.presubmit 'TestAuthZCheck|TestKubeInject|TestRevisionTags|TestUninstallByRevision|TestUninstallWithSetFlag|TestUninstallCustomFile|TestUninstallPurge'"
```

Comment:
I think it won't work. In master the script is called integ-suite-ocp.sh and in 1.24 it's called integ-suite-kind.sh
I think master's script accept a list of tests to skip.